doc/source/reference/api/grn_inspect.rst

```diff
@@ -0,0 +1,158 @@
+.. -*- rst -*-
+
+.. highlightlang:: none
+
+``grn_inspect``
+===============
+
+Summary
+-------
+
+There are two kind of groups to inspect :c:type:`grn_obj`. One is `grn_inspect`, The other is `grn_p` function.
+
+Here is the list of `grn_inspect` family. It sets inspected text into object.
+
+* :c:func:`grn_inspect()`
+* :c:func:`grn_inspect_indented()`
+* :c:func:`grn_inspect_limited()`
+* :c:func:`grn_inspect_name()`
+* :c:func:`grn_inspect_encoding()`
+* :c:func:`grn_inspect_type()`
+* :c:func:`grn_inspect_query_log_flags()`
+
+Here is the list of `grn_p` family. It prints inspected text into console.
+
+* :c:func:`grn_p()`
+* :c:func:`grn_p_geo_point()`
+* :c:func:`grn_p_ii_values()`
+
+Example
+-------
+
+Here is an example which inspects specified target object.
+
+.. code-block:: c
+
+   grn_obj buffer;
+   GRN_TEXT_INIT(&buffer, 0);
+   grn_inspect(&context, &buffer, obj);
+   /* equivalent to grn_p(ctx, obj); */
+   printf("inspected: <%.*s>\n", (int)GRN_TEXT_LEN(&buffer), GRN_TEXT_VALUE(&buffer));
```

Comment:
We should also write the result of executing this function.
Because, In my opinion, we easier to understand how to use this function.